kytos/utils/napps.py

```diff
@@ -185,29 +237,22 @@ def installed_dir(self):
         """Return the installed dir from current napp."""
         return self._installed / self.user / self.napp
 
-    def enable(self):
-        """Enable a NApp if not already enabled.
-
-        Raises:
-            FileNotFoundError: If NApp is not installed.
-            PermissionError: No filesystem permission to enable NApp.
-
-        """
-        core_napps_manager = CoreNAppsManager(base_path=self._enabled)
-        core_napps_manager.enable(self.user, self.napp)
-
     def is_enabled(self):
         """Whether a NApp is enabled."""
         return (self.user, self.napp) in self.get_enabled()
 
-    def uninstall(self):
+    def uninstall_remote(self):
         """Delete code inside NApp directory, if existent."""
-        if self.is_installed():
-            installed = self.installed_dir()
-            if installed.is_symlink():
-                installed.unlink()
+        uri = self._kytos_api + self._NAPP_UNINSTALL
+        uri = uri.format(self.user, self.napp)
+
+        try:
+            json.loads(urllib.request.urlopen(uri).read())
+        except urllib.error.HTTPError as exception:
+            if exception.code == HTTPStatus.BAD_REQUEST.value:
+                LOG.error("NApp is not uninstalled. Check the NApp list.")
```

Comment:
s/NApp is not uninstalled/Error uninstalling NApp/

Also, is there an error message that is returned by the server? If yes, please include the original message here.

Reply:
Ok. How about: "Check if the NApp is installed using NApp list."
I will open a new issue for Exception handling.